README.md

```diff
@@ -42,10 +42,11 @@
 
 This project allows you:
```

Comment:
- [ ] [optionakl] I would suggest you change this from this project to this milestone since the project has a larger scope than what you have listed.